src/Grafana/dashboards/aspnetcore.json

```diff
@@ -1281,16 +1292,17 @@
   "templating": {
     "list": [
       {
+        "allValue": ".*",
         "current": {},
         "datasource": {
           "type": "prometheus",
           "uid": "${DS_PROMETHEUS}"
         },
         "definition": "label_values(http_server_active_requests,job)",
         "hide": 0,
-        "includeAll": false,
+        "includeAll": true,
         "label": "Job",
-        "multi": false,
+        "multi": true,
```

Comment:
Enables mutli-select to be used, which is now feasible since all of the PromQL queries have been updated to support it.